### ISSUE TITLE
[ENHANCEMENT] Make English only version available

### ISSUE THREAD

**Issue by im-evo** [prusa3d/Prusa-Firmware-Buddy#3576]
**Printer type** - [MINI]
**Optional upgrades** - NA

**Is your suggested improvement related to an existing problem? Please describe.**
Hello, please add a build target for English-only firmware. At this time, it's only possible to install multi-language ones. Thanks!

**Describe the expected functionality**
An option to install English-only firmware.

**Comment by bkerler** [comment]
That's already possible by setting the additional cmake option "TRANSLATIONS_ENABLED" to "NO".

**Comment by im-evo** [comment]
> That's already possible by setting the additional cmake option "TRANSLATIONS_ENABLED" to "NO".

Nice, then the only thing missing is a build for releases so that a user is not forced to build it themselves or have a translation they will never use. 

It can also help track language preferences overall by the number of downloads.

**Comment by loglow** [comment]
Agreed. It is odd and confusing to have to pick one of six non-English languages to install. I actually came here because I was slightly confused about it.

**Comment by digiltd** [comment]
@loglow i agree, even though each file has the word `english` in it, when looking at them stacked in a folder list my brain just sees the differences i.e. the second country (stupid brain)

<img width="479" alt="Screenshot 2024-01-11 at 13 16 56@2x" src="https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/5680968/932752d1-94fb-46aa-a8fa-33f5aa06e3b6">
